src/components/tickets/components/Tickets.js

```diff
@@ -1,31 +1,15 @@
-import React, { useEffect, useState } from 'react';
-import { useDispatch, useSelector } from 'react-redux';
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
 import { Avatar, Button, Tag, Table, Row, Col, Menu } from 'antd';
 
-import {
-    getTickets,
-    getNewTickets,
-    getPendingTickets,
-    getResolvedTickets
-} from '../actions';
 import CreateTickets from './CreateTickets';
 import { ALL, PENDING, NEW, RESOLVED, HORIZONTAL } from '../constants';
 
 const menuItems = [ALL, PENDING, NEW, RESOLVED];
 const Tickets = ({ history }) => {
-    const dispatch = useDispatch();
+    const { user: { isAdmin } } = useSelector(state => state.user);
```

Comment:
create a `userSelector` to pull `state.user`

Reply:
The user in the state contains an object with another object that contains the users details.